src/index.html

```diff
@@ -1,17 +1,166 @@
+            name="name"
+            required
+            autocomplete="off">
+        </div>
+
+        <div class="age">
+          <label for="age">How old are You?</label>
+          <input
+            type="number"
+            id="age" name="age"
+            value="12"
+            min="12"
+            max="100"
+            required>
+        </div>
+        <div class="date">
+          <label for="birth_date">Full date of birth:</label>
+          <input
+            type="date"
+            id="birth_date"
+            name="birth_date">
+        </div>
+        <div class="check">
+          <label for="check_agreement">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            id="check_agreement"
+            required>
+        </div>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+
+        <div class="email">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            id="email"
+            name="email"
+            placeholder="email@example.com">
+        </div>
+        <div class="password">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            id="password"
+            name="password"
+            minlength="6"
+            maxlength="10">
+        </div>
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+
+        <div class="like_cats_yes">
+          Do you love cats?
+          <input
+            type="radio"
+            id="like_cats_yes"
+            name="like_cats"
+            value="yes">
+          <label for="like_cats_yes">Yes</label>
+          <input
+            type="radio"
+            id="like_cats_no"
+            name="like_cats"
+            value="no">
+          <label for="like_cats_no">No</label>
+        </div>
+        <div class="favorite_color">
+          <label for="favorite_color">What is your favorite color?</label>
+          <input
+            type="color"
+            name="favorite_color"
+            id="favorite_color">
+        </div>
+        <div class="go_to_bed">
+          <label for="go_to_bed">What time do you go to bed?</label>
+          <input
+            type="time"
+            id="go_to_bed"
+            name="go_to_bed">
+        </div>
+        <div class="cars">
+          <label for="cars">What are your favorite brend of cars?</label>
+          <select
+            name="cars"
+            id="cars"
+            multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </div>
+        <div class="rate">
+          <label for="rate">How do you rate our work?</label>
+          <input
+            type="range"
+            id="rate"
+            name="rate"
+            min="0" max="10"
+            value="0">
```

Comment:
```suggestion
          <input
            type="range"
            id="rate"
            name="rate"
            min="0" max="10"
            value="0"
          >
```
Закрывающую скобку располагай на одной вертикальной линии с открывающей

Reply:
сделал